Druid/DRCPSEdge.class.st

```diff
@@ -185,11 +191,21 @@ DRCPSEdge >> visitAnd: aDRAnd [
 ]
 
 { #category : #visiting }
-DRCPSEdge >> visitBitAnd: aDRBitAnd [
+DRCPSEdge >> visitBaseOffsetAddress: aDRBaseOffsetMemoryAddress [
+	"Not implemented"
 
 	^ nil
 ]
 
+{ #category : #visiting }
+DRCPSEdge >> visitBitAnd: aDRBitAnd [
+
+	^ (self operandConstraint: aDRBitAnd operand1) ifNotNil: [ :operandConstraint |
+		  aDRBitAnd operand2 isConstant ifTrue: [
+			  (operandConstraint bitAndConstraint: aDRBitAnd operand2 value) ifNotNil: [
+				  :constraint | self addConstraint: constraint to: aDRBitAnd result name ] ] ]
+]
```

Comment:
Rough

Reply:
First iteration: make it works.

Reply:
Probably it will not be necessary using PRE